src/main/java/com/example/auctionapp/dto/ProductDto.java

```diff
@@ -67,6 +80,8 @@ public ProductDto(Long id,
         this.feature = feature;
         this.userId = userId;
         this.bids = bids;
+        this.color = color;
+        this.size = size;
```

Comment:
`BaseResourceDTO` still contains local date times, it should have Timestamps

Reply:
These LocalDates on backend are made from timestamp that is sent from client. I use utility functions to convert it from timestamp to localdate, and vice versa.

Reply:
didnt see that this was dto, nvm my bad

Reply:
wait no, i didnt read this correctly. Why are we even using LocalDate time